accounting/src/delta/delta_data_collection/mod.rs

```diff
@@ -32,7 +45,6 @@ pub struct DeltaDataCollection<K: Ord, T> {
 }
 
 impl<K: Ord + Copy, T> DeltaDataCollection<K, T> {
-    #[cfg(test)]
     pub fn from_data<const N: usize>(data: [(K, DataDelta<T>); N]) -> Self {
```

Comment:
If you wanna keep this kind of initialization without it being exclusive for tests, perhaps it makes more sensible to implement the FromIterator trait for `(K, DataDelta<T>)` instead of this. This interface isn't ideal.